app/controllers/items_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ItemsController < ApplicationController
+  load_and_authorize_resource :wishlist
+  load_and_authorize_resource :item, :through => :wishlist
```

Comment:
Yeah, well. The item gets loaded, and authrized, and yet you still create your own in `#new` and `#create` and don't authorize the ones you've created manually. Use the `authorize!` manually.